### ISSUE TITLE
[Settings]: Voice Access users are not able to activate 'Add' button using 'Click Add' command as its Visible and Accessible name is different.

### ISSUE THREAD

**Issue by v-rpundir** [microsoft/terminal#13808]
### Windows Terminal version

1.15.2003.0

### Windows build number

10.0.25179.1000

### Other Software

**Test Environment:**
OS: Window 11 22H2 OS Build 25179.1000
App: Windows Terminal Preview
AT: Voice Access

### Steps to reproduce

**Repro Steps:**
1. Open Windows Terminal.
2. Open Voice Access AT.
3. Open Terminal Settings page.
4. Inside Defaults> Appearance page, Use 'Click Add ' command and observe the issue.

**User Experience:**
People with limited mobility who use Voice Access will have bad UX as they have to use show numbers command as control is not accessible by click command.

**Guideline Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/label-in-name

**Attachments:**
[Add.zip](https://github.com/microsoft/terminal/files/9394620/Add.zip)

Attachment Note: Not attached Code Snippet as not able to inspect Terminal Preview app with AI and Inspector tool.

### Expected Behavior

Visible name 'Add' should be included in the Accessible name Or it is recommended that accessible name should start with visible name.

### Actual Behavior

Visible (Add) and Accessible name (Create Appearance) is different for 'Add' button inside 'Defaults> Appearance' page.

**Observation:** Voice Access users can access buggy control using 'Show numbers', 'Show Grid' and keyboard shortcut commands.

To learn more about Voice Access and how an application can support its different modes, Refer https://www.osgwiki.com/wiki/VoiceAccess_Testing

Voice Access Commands page: https://sway.office.com/Q5JvoIGBI5glyoKw?ref=Link

**Comment by Priyanka-Chauhan123** [comment]
@zadjii-msft Is there any ETA to get this bug fixed?

**Comment by zadjii-msft** [comment]
@Priyanka-Chauhan123 Not immediately - it's in our current release milestone, but I don't have a more accurate ETA than that.

**Comment by Priyanka-Chauhan123** [comment]
@zadjii-msft Actually this is the only bug which is impacting our product grade and making it as D compliant.
